src/main/java/com/swordglowsblue/artifice/api/builder/data/dimension/DimensionTypeBuilder.java

```
@@ -54,6 +54,14 @@ public DimensionTypeBuilder shrunk(boolean shrunk) {
      * @return
      */
     public DimensionTypeBuilder ambientLight(float ambientLight) {
+        if (ambientLight < 0.0F) {
+            ambientLight = 0.0F;
+            System.out.println("Ambient light can't be under 0.0, setting it to 0.0");
```

I think you should throw instead of setting a default value